src/vmm/src/rpc_interface.rs

```diff
@@ -126,88 +129,71 @@ pub enum VmmAction {
 }
 
 /// Wrapper for all errors associated with VMM actions.
-#[derive(Debug, derive_more::From)]
+#[derive(Debug, thiserror::Error, derive_more::From)]
 pub enum VmmActionError {
     /// The action `SetBalloonDevice` failed because of bad user input.
+    #[error("{0}")]
```

Comment:
In these cases could we simply include the comment?
```suggestion
    /// The action `SetBalloonDevice` failed because of bad user input.
    #[error("The action `SetBalloonDevice` failed because of bad user input: {0}")]
```

Reply:
I had done this initially, but there are tests that depend on the error message. I don't want to fix all these in this PR.

Reply:
Would it be worth creating an issue for this, maybe ``Ensure error types have descriptive `Display` implementations``?

Reply:
I guess that this: https://github.com/firecracker-microvm/firecracker/issues/3278 would address it.

Reply:
I think https://github.com/firecracker-microvm/firecracker/issues/3278 is different, this type already implements `thiserror::Error` so the issue wouldn't entail any action here.

Reply:
Ah sorry, I got confused with the other case, where included types did not implement `thiserror::Error`. So, if you check the message printed by the including type is already quite descriptive I think.